inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -3579,3 +3579,115 @@ INSTANTIATE_TEST_CASE_P(fusings_gpu, deconv_scale_actv_quant_u8_eltw_scale_actv_
+    }
+};
+
+class eltwise_quantize : public EltwiseFusingTest {};
+TEST_P(eltwise_quantize, basic) {
+    auto p = GetParam();
+    create_topologies(input_layout("input", get_input_layout(p)),
+                      data("input2", get_mem(get_input_layout(p))),
+                      eltwise("eltwise", {"input", "input2"}, p.mode),
+                      data("in_lo", get_mem(get_single_element_layout(p), min_random, 0)),
+                      data("in_hi", get_mem(get_single_element_layout(p), 1, max_random)),
+                      data("out_lo", get_mem(get_single_element_layout(p), 0)),
+                      data("out_hi", get_mem(get_single_element_layout(p), 255)),
+                      quantize("quantize", "eltwise", "in_lo", "in_hi", "out_lo", "out_hi", 256, data_types::u8),
+                      reorder("out", "quantize", p.default_format, data_types::f32));
+
+    tolerance = 1.f;
+    execute(p);
+}
+
+class eltwise_quantize_int8 : public EltwiseFusingTest {};
+TEST_P(eltwise_quantize_int8, basic) {
+    auto p = GetParam();
+    create_topologies(input_layout("input", get_input_layout(p)),
+                      data("input2", get_mem(get_input_layout(p))),
+                      eltwise("eltwise", {"input", "input2"}, p.mode, data_types::f32),
+                      data("in_lo", get_mem(get_single_element_layout(p), min_random, 0)),
+                      data("in_hi", get_mem(get_single_element_layout(p), 1, max_random)),
+                      data("out_lo", get_mem(get_single_element_layout(p), 0)),
+                      data("out_hi", get_mem(get_single_element_layout(p), 255)),
+                      quantize("quantize", "eltwise", "in_lo", "in_hi", "out_lo", "out_hi", 256, data_types::u8),
+                      reorder("out", "quantize", p.default_format, data_types::f32));
+
+    tolerance = 1.f;
+    execute(p);
+}
+
+INSTANTIATE_TEST_CASE_P(fusings_gpu,
+                        eltwise_quantize,
+                        ::testing::ValuesIn(std::vector<eltwise_test_params>{
+                            eltwise_tests(CASE_ELTWISE_FP32_1),
+                            eltwise_tests_fp32(CASE_ELTWISE_FP32_1),
+                            eltwise_tests(CASE_ELTWISE_FP32_2),
+                            eltwise_tests_fp32(CASE_ELTWISE_FP32_2),
+                            eltwise_tests(CASE_ELTWISE_FP32_3),
+                            eltwise_tests_fp32(CASE_ELTWISE_FP32_3),
+                        }), );
+
+INSTANTIATE_TEST_CASE_P(fusings_gpu,
+                        eltwise_quantize_int8,
+                        ::testing::ValuesIn(std::vector<eltwise_test_params>{
+                            eltwise_tests(CASE_ELTWISE_I8_1),
+                            eltwise_tests(CASE_ELTWISE_I8_2),
+                            eltwise_tests(CASE_ELTWISE_U8_1),
+                            eltwise_tests(CASE_ELTWISE_U8_2),
+                        }), );
```

Comment:
Could you add some test where eltwise has inputs in two different types (ex. i8 + fp32, fp32 + fp16)?
There are some issues in fsv16 kernel that such test could have caught.